MIGRATION-4.0.0.MD

```diff
@@ -0,0 +1,83 @@
+#in progress
+
+# Kakunin - automated testing framework
+#### Migration to 4.0.0 version
+
+What needs to be done:
```

Comment:
It's a confusion for me. I think it should be like:
` To do the migration:
 1. Add for kakunin.conf.js info that imageComparator.baseline is required 
`